src/Hosting/Hosting/test/WebHostTests.cs

```diff
@@ -313,7 +312,6 @@ public async Task WebHostStopAsyncCanBeCancelledEarly()
         }
 
         [ConditionalFact]
-        [QuarantinedTest]
```

Comment:
@BrennanConroy @JunTaoLuo looks like you added these tests. Let me know if you're okay with unquarantining these.

Reply:
These were marked quarantined when they were added to Helix, so probably fine to un-quarantine if they are passing.